### DISCUSSION TITLE
Passing parameters in alert rules

### DISCUSSION THREAD

**Discussion by jayluo667** [ccfos/nightingale#1278] [Q&A]
版本：5.14.1，告警规则使用ibex作为回调
情景描述：
     端口监控告警（某个服务挂了），想通过ibex告警自愈起起来，但是需要获取到某个标签的值作为 ibex脚本的传入参数
问题：
     目前看到5.14.1版本已经支持 告警信息 和 备注 配置变量{{$labels.ident}} 这类，告警自愈-自愈脚本里虽然支持传参（但不支持配置变量吧）请问大佬目前有解决的方式吗？

**Comment by UlricQin** [comment]
目前做不到变量的联动。默认的处理逻辑是，告警之后自动去告警的机器上跑脚本，这个在B站视频上有介绍

**Reply by jayluo667** [reply]
感谢百忙中抽空回答，告警之后自动去告警的机器上跑脚本-->这个功能已学习并配置了少部分回调，比如像告警categraf、MySQL等挂了，配置回调然后启动起来，这类的特点是确定性参数，目前ibex适用于这类操作。
